src/pages/PageUsingComponents.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { SplitClient, SplitTreatments } from '@splitsoftware/splitio-react';
+import { feature_1, feature_2, feature_3 } from '../sdkConfig';
+
+/* This example showcasts SplitClient and SplitTreatments components */
+
+const Loading = () => {
+  return <div>Loading SDK...</div>
+}
+
+export default () => {
+  return (
+    <main>
+
+      <SplitTreatments names={[feature_1]} >
+        {({ treatments }) => {
+          return (
+            <div className="App-section">
+              <h4>{`Split: ${feature_1}`}</h4>
+              <p>{`Treatment value: ${treatments[feature_1].treatment}`}</p>
+            </div>
+          )
+        }}
+      </SplitTreatments>
+
+      <SplitClient splitKey="other_user" >
+        <SplitTreatments names={[feature_2, feature_3]} >
+          {({ treatments, isReady }) => {
+            return isReady ? (
```

Comment:
Either add isReady to the previous one or make a mention there. I wouldn't want people getting confused and thinking isReady is only available because it is under a new client. 

If it was me I would add it to the previous one, say it's tied to the main client. Then on this one, I would say the isReady will depend on the new client for other_user.

Reply:
Decided to keep the same code but with additional comments to explain the importance of using or not the SDK status props.